modules/tensor_mechanics/src/kernels/StiffPropDamping.C

```diff
@@ -0,0 +1,50 @@
+//* This file is part of the MOOSE framework
```

Comment:
Stiffness proportional damping already exists in MOOSE and I don't think this is needed. Please see https://github.com/idaholab/moose/blob/next/modules/tensor_mechanics/src/kernels/DynamicStressDivergenceTensors.C 
This formulations includes both HHT integration (alpha) and stiffness damping (zeta). We typically use zero for alpha and this formulation should reduce to pure stiffness damping.

Reply:
Hi @cbolisetti, I did a simplification here:

```
residual +=
        _stress[_qp].row(_component) * _grad_test[_i][_qp] *
            (1.0 + _zeta[_qp] / _dt) -
        ((1.0) * _zeta[_qp] / _dt) * _stress_old[_qp].row(_component) *
            _grad_test[_i][_qp] 
```
```
residual +=   _stress[_qp].row(_component) * _grad_test[_i][_qp] + _zeta[_qp] / _dt * ( _stress[_qp].row(_component) * _grad_test[_i][_qp]  -  _stress_old[_qp].row(_component) * _grad_test[_i][_qp] )
```

The question that may exist is that in an explicit solver, by the time this kernel gets executed, `_stress[_qp]` and `_stress_old[_qp]` is the same thing because the displacement hasn't been updated yet. That's why I'm using `_stress[_qp] - _stress_older[_qp]` (or actually `_stress_old[_qp] - _stress_older[_qp]` will also do the trick) to make the damping effect